zivid_samples/launch/sample.launch

```diff
@@ -1,6 +1,8 @@
 <launch>
+    <arg name="type"/>
+    <arg name="name" default=""/>
```

Comment:
When empty string is used, it will end up using a default node name which is specified in the sample itself, like 
`rospy.init_node("sample_capture_2d_py", anonymous=True)`